log/error_test.go

```diff
@@ -38,7 +30,7 @@ func TestFormatErrorsFunc(t *testing.T) {
 		So(origin.File, ShouldEndWith, "log.go/log/error_test.go")
 
 		// If this test fails, check the `errEventData := Error(err).(*EventErrors)` line is still line 34!
-		So(origin.Line, ShouldEqual, 34)
+		So(origin.Line, ShouldEqual, 26)
```

Comment:
Whats with the magic number of 26
Shouldn't there be const strings and taking lengths of those ?

Reply:
This is the line number of when an event is thrown, as we update this file the test might move line number and this is reflected in this value changing - hence a previous commit of mine was to move this test to the top of the test (less likely to change).

Reply:
I do need to update the comment